tests/unit/bokeh/embed/test_bundle.py

```diff
@@ -38,61 +38,57 @@
 ABSOLUTE = Resources(mode="absolute")
 SERVER = Resources(mode="server")
 
-@pytest.fixture
-def test_plot() -> None:
+def plot() -> models.Plot:
     from bokeh.plotting import figure
-    test_plot = figure()
-    test_plot.scatter([1, 2], [2, 3])
-    return test_plot
+    plot = figure()
+    plot.scatter([1, 2], [2, 3])
+    return plot
 
-@pytest.fixture
-def test_glplot() -> None:
+def glplot() -> models.Plot:
     from bokeh.plotting import figure
-    test_glplot = figure(output_backend="webgl")
-    test_glplot.scatter([1, 2], [2, 3])
-    return test_glplot
-
-@pytest.fixture
-def test_table() -> None:
-    from bokeh.models import DataTable
-    test_table = DataTable()
-    return test_table
-
-@pytest.fixture
-def test_widget() -> None:
-    from bokeh.models import Button
-    test_widget = Button()
-    return test_widget
-
-@pytest.fixture
-def test_mathtext() -> None:
-    from bokeh.models import TeX
-    test_mathtext = TeX()
-    return test_mathtext
-
-@pytest.fixture
-def test_mathstring_axis_label() -> None:
-    from bokeh.models import LinearAxis
-    test_mathstring = LinearAxis(axis_label = "$$sin(x)$$")
-    return test_mathstring
-
-@pytest.fixture
-def test_mathstring_axis_label_with_parenthesis() -> None:
-    from bokeh.models import LinearAxis
-    test_mathstring = LinearAxis(axis_label = r"\(sin(x)\)")
-    return test_mathstring
-
-@pytest.fixture
-def test_mathstring_major_label_overrides() -> None:
-    from bokeh.models import LinearAxis
-    test_mathstring = LinearAxis(major_label_overrides = {0: r"\[sin(x)\]"})
-    return test_mathstring
-
-@pytest.fixture
-def test_plaintext() -> None:
-    from bokeh.models import PlainText
-    test_plaintext = PlainText("$$sin(x)$$")
-    return test_plaintext
+    glplot = figure(output_backend="webgl")
+    glplot.scatter([1, 2], [2, 3])
+    return glplot
+
+def table() -> models.DataTable:
+    table = models.DataTable()
+    return table
```

Comment:
I have a vague recollection that the fixtures imports were there to defer the `bokeh.models` import for some reason, but I don't recall any detail, and maybe it's not relevant any longer, if it ever was. 

In any case, as long as we are simplifying (here and below):

```suggestion
    return models.DataTable()
```